10_SL1/errors.yaml

```diff
@@ -264,6 +264,12 @@ Errors:
   id: "UV_INTENSITY_DEVIATION_TOO_HIGH"
   approved: true
 
+- code: "10320"
+  title: "BOOSTER BOARD PROBLEM"
+  text: "Communication with Booster board failed. Contact our support."
```

Comment:
Nemelo by v ramci prvni pomoci byt aby si clovek zkontroloval kabely atd a az pak otravoval support?

Reply:
To mu snad může poradit support nebo ta webová stránka s popisem ne?

Reply:
Tak muze, ale kdyz jsem koukal na ostatni error kody, tak u sebe mivaji nejake rozreseni rovnou https://github.com/prusa3d/Prusa-Error-Codes/blob/master/10_SL1/errors.yaml#L39 Proc by mel clovek nacitat QR kod, kdyz mu ta stranka nerekne nic vic nez ze ma zkontrolovat kabely a pokud to nepomuze, tak jit za supportem?

Reply:
Je tu celé oddělení na to co se má něco nemá psát uživateli. Mě stačí, že chyba existuje a má nějaký text, stejně se to bude několikrát měnit kvůli angličtině, slovosledu a podobně...

Reply:
Líbí? :-) Jsem zvědavý, kolik uživatelů skončí v nekonečné hře kontroly kabelů a restartování tiskárny :-D